src/nbl/asset/utils/waveContext.h

```diff
@@ -97,6 +102,34 @@ struct preprocessing_hooks final : public boost::wave::context_policies::default
             m_pragmaStage = found->second;
             return true;
         }
+        
+        if (strcmp(optionStr, "dxc_compile_flags") == 0 && hash_token_occurences == 1) {
```

Comment:
you need to spam me with an ERROR or WARNING if you encounter `dxc_compile_flags` and the hast token occurence counter isn't 1,  instead of silently skipping